app_dart/test/request_handlers/append_log_test.dart

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:typed_data';
+
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+import 'package:cocoon_service/src/request_handlers/append_log.dart';
+import 'package:cocoon_service/src/service/stackdriver_logger.dart';
+
+import '../src/datastore/fake_cocoon_config.dart';
+import '../src/request_handling/fake_authentication.dart';
+
+void main() {
+  group('AppendLog', () {
```

Comment:
I believe append_log_test.dart was missing before this PR. It would be good to add tests for existing features in append_log.dart. Thanks.

Reply:
Added!